src/rules/flake8_use_pathlib/fixes.rs

```diff
@@ -0,0 +1,68 @@
+use rustpython_ast::{Expr, ExprKind};
+
+use crate::ast::helpers;
+use crate::ast::types::Range;
+use crate::autofix::apply_fix;
+use crate::checkers::ast::Checker;
+use crate::fix::Fix;
+use crate::registry::DiagnosticKind;
+use crate::source_code::Locator;
+
+pub fn pathlib_fix(
+    checker: &mut Checker,
+    diagnostic: &DiagnosticKind,
+    expr: &Expr,
+    parent: Option<&Expr>,
+) -> Option<Fix> {
+    // TODO: if pathlib not imported, then add to imports. Remove os import if not referenced
```

Comment:
I think we should guard all of these to only autofix if `Path` is imported and available as `Path`. (Are we already doing that? I admittedly haven't read the PR through fully.)

Reply:
Just my 2 cents.
I think `Path` could be somewhat ambiguous. 
I tend always to use the fully qualified `pathlib.Path` version.

I've been bitten at least once before by seeing a `Path` object, assuming it's a `pathlib.Path` object only to find out it's something like a `glom.Path`.
https://glom.readthedocs.io/en/latest/api.html#glom.Path
There's a least one other somewhat popular library that uses `Path` object but I'm having trouble remembering it. 😅 

Very excited about these auto-fixes BTW 🚀 ❤️

Reply:
Yeah we have the ability to ensure that whatever's referenced is indeed `pathlib.Path`, even in the following cases:

```py
import pathlib
pathlib.Path  # OK!

from pathlib import Path
Path  # OK!

from pathlib import Path as P
P  # OK!

Path = 1 + 2
Path  # Not OK!
```

So we _should_ be able to support and preserve whatever import style is used.

Reply:
Will introduce the guarding, great suggestion. Thank you.

Related, but could also open a dedicated issue for them:
- How is the user notified of this behaviour? I suggest expanding the autofix documentation to include these kinds of directions.

- Say the user would like to autofix the code base, should they first do a manual pass through the code to add all imports? This seems suboptimal from a user perspective. There is a similar challenge with future annotations, with the difference that they become obsolete when the python target increases.

Reply:
(We have some logic for this in `src/rules/pylint/rules/use_sys_exit.rs`, hopefully it can be generalized.)

Hmm. I'm not sure I have great answers. The best long-term solution, probably, would be to fix #835 so we _can_ add the import as part of the fix.

Reply:
(Still working on this btw)